.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+
+python:
+  - 3.6
+
+dist: xenial
+sudo: false
+
+cache: pip
+
+install:
+  - pip install --upgrade pycodestyle
```

Comment:
Travis runs `pip install -r requirements` by default on python projects, so you should maybe evaluate again what's overkill here or not.